libraries/AP_Scheduler/AP_Scheduler.cpp

```diff
@@ -412,7 +412,7 @@ size_t AP_Scheduler::task_info(char *buf, size_t bufsize)
     // a header to allow for machine parsers to determine format
     int n = hal.util->snprintf(buf, bufsize, "TasksV1\n");
 
-    if (n <= 0) {
+    if (n < 8) {
```

Comment:
Perhaps:
```
    const char *header = "TasksV1\n";
    int n = hal.util->snprintf(buf, bufsize, header);
    if (n < strlen(header)) {
```

to avoid the constant.

@andyp1per for visibility

Reply:
Works for me, although the chances of us writing only part of the header seem pretty remote to me

Reply:
@peterbarker san. I agree with the suggestion.